src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -100,5 +132,33 @@ public void AddTagHelperAttribute([NotNull] string name, object value)
         {
             AllAttributes.Add(name, value);
         }
+
+        /// <summary>
+        /// Executes the child content asynchronously.
+        /// </summary>
+        /// <returns>A task that indicates when child content execution has completed.</returns>
```

Comment:
I said "or some such" and don't care much about the word "renders here.  the important bit is that the `Task` doesn't indicate anything about completion directly -- that makes it sounds like it's a `Task<bool>` where the `bool` indicates the children have executed.  describe what the `Task` actually does or has occurred before it completes.